.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
-  - "1.11"
-  - "1.12"
+  - "1.11.x"
```

Comment:
Because of https://github.com/golang/go/issues/30446

(TIL I learned travis takes `.x`)